src/mol-plugin-ui/structure/quick-styles.tsx

```diff
@@ -31,110 +33,131 @@ export class StructureQuickStylesControls extends CollapsableControls {
     }
 }
 
-export class QuickStyles extends PurePluginUIComponent {
-    async default() {
-        const { structures } = this.plugin.managers.structure.hierarchy.selection;
-        const preset = this.plugin.config.get(PluginConfig.Structure.DefaultRepresentationPreset) || PresetStructureRepresentations.auto.id;
-        const provider = this.plugin.builders.structure.representation.resolveProvider(preset);
-        await this.plugin.managers.structure.component.applyPreset(structures, provider);
 
-        this.plugin.managers.structure.component.setOptions(PD.getDefaultValues(StructureComponentManager.OptionsParams));
+const QuickStyleParams = {
+    preset: PD.Select('', [['', ''], ['default', 'Default'], ['cartoon', 'Cartoon'], ['spacefill', 'Spacefill'], ['surface', 'Surface']] as const, { isHidden: true }),
+    stylized: PD.Boolean(false, { description: 'Toggles stylized appearance (outline, occlusion effects, ignore-light representation parameter). Does not affect representation type.' }),
+};
 
-        if (this.plugin.canvas3d) {
-            const p = PD.getDefaultValues(PostprocessingParams);
-            this.plugin.canvas3d.setProps({
-                postprocessing: { outline: p.outline, occlusion: p.occlusion }
-            });
-        }
+type QuickStyleProps = PD.Values<typeof QuickStyleParams>;
+
+interface QuickStylesState {
+    props: QuickStyleProps,
+    busy: boolean,
+}
+
+
+export class QuickStyles extends PurePluginUIComponent<{}, QuickStylesState> {
+    state: QuickStylesState = { props: { preset: '', stylized: false }, busy: false };
+
+    componentDidMount() {
+        const invalidatePreset = () => this.setState(old => ({ props: { ...old.props, preset: '' } }));
+        this.subscribe(this.plugin.state.data.events.cell.created, invalidatePreset);
+        this.subscribe(this.plugin.state.data.events.cell.stateUpdated, invalidatePreset);
```

Comment:
Can you filter this by `isAnimating` and I think `changed` event should be enough? Something like:

```ts
       this.subscribe(plugin.state.data.events.changed, e => {
            if (e.inTransaction || plugin.behaviors.state.isAnimating.value) return;
            this.sync(true);
        });

        this.subscribe(plugin.behaviors.state.isAnimating, isAnimating => {
            if (!isAnimating && !plugin.behaviors.state.isUpdating.value) this.sync(true);
        });
```

The constant UI updates could reduce the FPS quite a bit during animation.

Reply:
OK, i changed it.